.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Pull Requests
+
+on:
+  pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
```

Comment:
```suggestion
        node-version: [18.x, 20.x, 21.x]
```

Reply:
@alexs-mparticle why not `node-latest` as suggested elsewhere?

Reply:
My view is that for Pull requests, we should know if the code will work for both legacy and upcoming releases so we can act on it while we have context. For releases, we just need node to ship code, so we can use whatever node version is supported by the ci/cd pipeline.